packages/website/docs/_samples/main/Menu/DynamicallyAddedItems/main.js

```diff
@@ -0,0 +1,64 @@
+import "@ui5/webcomponents/dist/Menu.js";
+import "@ui5/webcomponents/dist/MenuItem.js";
```

Comment:
import Button otherwise you depend on side effect that it comes from the menu.

clear imports of the icons that are not used